include/torch-mlir/Dialect/Torch/Transforms/Passes.td

```diff
@@ -151,7 +151,11 @@ def InlineGlobalSlots : Pass<"torch-inline-global-slots", "ModuleOp"> {
 
 def ReduceOpVariants : Pass<"torch-reduce-op-variants", "func::FuncOp"> {
   let summary = "Reduces variants of ops to a smaller set of ops.";
-  let constructor = "mlir::torch::Torch::createReduceOpVariantsPass()";
+  let constructor = "mlir::torch::Torch::createReduceOpVariantsPass(\"\")";
```

Comment:
nit: can you add `/*extraLibrary=*/` to the constructor argument to be consistent with other constructors in this file?

Same comment applies to changes below